README.md

```diff
@@ -61,9 +60,5 @@ MIT
 [npm-url]: https://npmjs.com/package/vite
 [node-img]: https://img.shields.io/node/v/vite.svg
 [node-url]: https://nodejs.org/en/about/releases/
-[unix-ci-img]: https://circleci.com/gh/vitejs/vite/tree/main.svg?style=shield
-[unix-ci-url]: https://app.circleci.com/pipelines/github/vitejs/vite?branch=main
-[windows-ci-img]: https://ci.appveyor.com/api/projects/status/0q4j8062olbcs71l/branch/main?svg=true
-[windows-ci-url]: https://ci.appveyor.com/project/yyx990803/vite/branch/main
 [discord-img]: https://img.shields.io/badge/chat-discord-blue?style=flat&logo=discord
 [discord-url]: https://chat.vitejs.dev
```

Comment:
This entire section looks like unused reference-style links to me 👀

Reply:
Yeah I also was curios about it :thinking:

Reply:
Didn't touched it, cause it is unrelated to the PR :thinking: